.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,52 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
```

Comment:
When we decide where & how to host shiny apps long term, we can add some deployment options here.